templates/backend/contribution/create.php

```diff
@@ -59,9 +59,14 @@
 						<img src="<?php echo esc_url( $action_icons['option_info'] ); ?>">
 					</button>
 				</label>
-				<button class="rev-gen-contribution-main-generate-button">
-					<?php esc_html_e( 'Generate and copy code', 'revenue-generator' ); ?>
-				</button>
+				<div id="rg_contribution_generate" class="rev-gen-contribution-main-button">
+					<button class="rev-gen-contribution-main-generate-button">
+						<?php esc_html_e( 'Generate and copy code', 'revenue-generator' ); ?>
+					</button>
+					<a target="_blank" href="https://wordpress.com/support/shortcodes/" id="rev-gen-contribution-help-generate" class="rev-gen-contribution-main--help-button">
+							<img src="<?php echo esc_url( $action_icons['option_info'] ); ?>">
```

Comment:
Can we remove this extra indentation?
```suggestion
						<img src="<?php echo esc_url( $action_icons['option_info'] ); ?>">
```